loops/csharp/csharp.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Please use .net 9

Reply:
I didn't realize that .NET 9 is already in Gentoo, it is masked, but I'm installing it right now to update this :)